tests/unit/test_workspace_templates.py

```
@@ -0,0 +1,131 @@
+        try:
+            updated = client.workspace_templates.work_items.update(
+                workspace_slug,
+                created.id,
+                UpdateWorkItemTemplate(description="Updated description"),
+            )
+            assert updated.id == created.id
+            assert updated.description == "Updated description"
+        finally:
+            try:
+                client.workspace_templates.work_items.delete(workspace_slug, created.id)
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)